dstyle.dd

```diff
@@ -25,6 +25,29 @@ $(UL
     $(LI Use spaces instead of hardware tabs.)
 
     $(LI Each indentation level will be four columns.)
+
+    $(LI Braces should be on their own line. There are a few exceptions to this
```

Comment:
If you look a the original, all I did was remove the redundant wording, I did not actually change the semantic content.